Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,81 @@
+#include "Person.h"
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+
+Person::Person()
+{
+    name_="";
+    age_=new int;
+    
+}
+
+
```

Comment:
Remove extra whitespace between all your functions.

Reply:
I thought we should leave 1-2 whitespaces between functions. Should I completely remove them?

Reply:
Just leave 1 please